### PR TITLE
chore: Document workaround for sending cookies services to datalayer

### PR DESCRIPTION
**Pull request by Kagaminara**
This MR document an issue about sending cookie services to the datalayer.

[Analytics page](https://deploy-preview-681--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/getting-started#retrieve-authorized-cookie-services-in-analytics)\
[GTM mention](https://deploy-preview-681--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/plugins#retrieve-authorized-cookie-services-in-gtms-datalayer)

### CODE REVIEW

File: versioned_docs/version-2.x/advanced/analytics/plugins.mdx [hunk 1]
```diff
@@ -366,3 +366,24 @@ Feel free to browse
 contribute to them if you have any ideas! 💡
 
 :::
+
+## Known issue
```
Comment (real34) [inline versioned_docs/version-2.x/advanced/analytics/plugins.mdx:370]: **issue:** several issues here IMO
1. this isn't an issue per se
2. datalayer is specific to GTM, so it shouldn't be mentioned for this part
3. it is more related to the signature of `settings()`

**suggestion:** rework this MR to highlight this in https://deploy-preview-681--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/getting-started#configure-analytics and maybe add a note to the new section from GTM's example

**Review by real34 — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.x/advanced/analytics/plugins.mdx (hunk 1 above)
Comment (Kagaminara) [reply versioned_docs/version-2.x/advanced/analytics/plugins.mdx]: I added a section in getting started to mention the workaround, removed the issue mention, and updated GTM's documentation to mention it.

Please let me know if the wording isn't right still.

**Review by paul-vd — APPROVED** [review]

File: versioned_docs/version-2.x/advanced/analytics/getting-started.mdx [hunk 2]
```diff
@@ -94,6 +94,30 @@ module.exports = {
 };
 ```
 
+### Retrieve authorized cookie services in analytics
+
+Currently in Front-Commerce, in order to send authorized cookie services to
+analytics services, those services also need to be defined in
```
Comment (real34) [inline versioned_docs/version-2.x/advanced/analytics/getting-started.mdx:100]: **nitpick:** Its not very clear, as it feel that they're "automatically sent" to analytics services. They're not. So far, only GTM supports this if I'm not mistaken.

**suggestion:** mention where it is sent in Analytics plugins

```suggestion
In Front-Commerce, to allow authorized cookie services to
be injected in analytics services `settings(authorization, otherAuthorizations)` callback (as its 2nd parameter), those services also need to be defined in
```

File: versioned_docs/version-2.x/advanced/analytics/plugins.mdx [hunk 3]
```diff
@@ -344,6 +344,12 @@ load / remove (depending on the `userConsents` value). Here is an example:
 
 </Figure>
 
+#### Retrieve authorized cookie services in GTM's datalayer
+
+If your project needs to retrieve the authorized cookies services in GTM's
+datalayer, in example for triggering other analytics services, you can do so by
+[declaring them in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).
```
Comment (real34) [inline versioned_docs/version-2.x/advanced/analytics/plugins.mdx:351]: **nitpick:** it feels like a complement to the previous explanation

**suggestion:** 
```suggestion
Please note that to retrieve the authorized cookies services in GTM's
datalayer, services must be [declared in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).
```

**Review by real34 — APPROVED** [review]

**Pull request merged**